cli/index.ts

```diff
@@ -66,21 +66,20 @@ const credentialsOption = {
   }
 };
 
+const warehouseOption = {
+  name: "warehouse",
+  option: {
+    describe: "The project's data warehouse type.",
+    choices: ["bigquery", "redshift", "snowflake"]
```

Comment:
Should we generate this from WarehouseTypes ?

Reply:
Good idea, done